### PR TITLE
[MM-28140] e2e/system_console/customization: add T1024

### PR DESCRIPTION
**Pull request by mgdelacroix**
#### Summary
Adds T1024: Customization Custom Branding - Can change name and desc with custom branding set to false

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-28140

### CODE REVIEW

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js [hunk 1]
```diff
@@ -396,6 +396,31 @@ describe('Customization', () => {
             expect(config.TeamSettings.EnableCustomBrand).to.equal(false);
         });
     });
+
+    it('T1024 - Can change name and desc with Custom Branding set to false', () => {
```
Comment (mgdelacroix) [inline e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:400]: Wondering if this should go in another file, like `e2e/cypress/integration/system_console/site_configuration/customization_spec.js`, to respect TM4J folder structure

/cc @saturninoabril

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js [hunk 2]
```diff
@@ -396,6 +396,34 @@ describe('Customization', () => {
...
+        // # Make sure necessary field is false
+        cy.apiUpdateConfig({TeamSettings: {EnableCustomBrand: false}});
+        cy.reload();
+
+        // * Verify that setting is visible and matches text content
+        cy.findByTestId('TeamSettings.SiteNamelabel').scrollIntoView().should('be.visible').and('have.text', 'Site Name:');
+
+        // # Update both Site Name and Description to store test values
+        const siteName = 'Mattermost_Text';
```
Comment (isacikgoz) [inline e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:409]: Nit: did you mean `test`?
```suggestion
        const siteName = 'Mattermost_Test';
```

**Review by isacikgoz — APPROVED** [review]
LGTM, just a non blocking nit.

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js (hunk 2 above)
Comment (mgdelacroix) [reply e2e/cypress/integration/system_console/ui_and_api/customization_spec.js]: Right! Good catch, thanks! :)

File: e2e/cypress/integration/system_console/ui_and_api/customization_spec.js (hunk 1 above)
Comment (saturninoabril) [reply e2e/cypress/integration/system_console/ui_and_api/customization_spec.js:400]: @mgdelacroix Yes, please. Thanks for asking.

Also, please include full test ID - `MM-T1024`
```suggestion
    it('MM-T1024 - Can change name and desc with Custom Branding set to false', () => {
```

**Review by saturninoabril — CHANGES_REQUESTED** [review]
@mgdelacroix Thanks! Tested and passed. LGTM, except per comment.

**Review by saturninoabril — APPROVED** [review]
Thanks @mgdelacroix. LGTM!

File: e2e/cypress/integration/system_console/site_configuration/customization_spec.js [hunk 3]
```diff
@@ -0,0 +1,59 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Stage: @prod
```
Comment (saturninoabril) [inline e2e/cypress/integration/system_console/site_configuration/customization_spec.js:10]: Please remove. We'll add it back when we found this spec to be stable for few Cypress runs.  Good catch Lindy!
```suggestion
```

**Review by lindy65 — APPROVED** [review]
Thanks @mgdelacroix - looks good to me 👍

**Pull request merged**